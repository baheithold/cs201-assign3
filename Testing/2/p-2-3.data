3 0 382 ; 5 5 112 ; 7 5 130 ; 5 2 287 ; 0 5 69 ; 
3 4 435 ; 5 5 115 ; 2 3 402 ; 0 0 445 ; 2 5 366 ; 
2 3 477 ; 7 7 468 ; 
