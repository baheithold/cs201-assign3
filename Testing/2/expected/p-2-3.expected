0: 3
1: 0(3)382 4(3)435
2: 5(0)69
3: 2(5)287 7(5)130
weight: 1303
